editor/src/components/canvas/controls/select-mode/padding-value-indicator.tsx

```diff
@@ -38,7 +46,7 @@ export const PaddingValueIndicator = controlForStrategyMemoized<PaddingValueIndi
             color: 'white',
           }}
         >
-          {actualPaddingValue}
+          {actualPaddingValue.value + (actualPaddingValue.unit ?? 'px')}
```

Comment:
This appears to be inlining that same css number printing code

Reply:
thanks! I've replaces this with a call to `printCSSNumber`